rfcs/0019-storing-client-data.md

```diff
@@ -421,107 +421,11 @@ The API process will also make subscription to changes in the daemon and changes
-    | Unknown -> (false, Unknown)
-    | Added -> (false, Scheduled)
-    | Dropped -> (false, Failed)
-  | results ->
-      let transaction_status =
-        if List.for_all results ~f:(fun (_, _, `Cosensus_status consensus_status_id)  ->
-        let status = Consensus_status.of_int status in
-         Consensus_status.equal status Failed) then
-          (false, Failed)
-      else
-         let confirmed_result = List.find_map results ~f:(fun (_, `Ledger_proof_nonce nonce, `Cosensus_status consensus_status_id)  ->
-          let status = Consensus_status.of_int status in
-          Option.some_if (Consensus_status.equal status Confirmed) nonce)  |>
-        Option.map  
-          ~f:(fun ledger_proof_nonce ->
-            (is_transaction_snarked best_tip_breadcrumb ledger_proof_nonce, true)
-          ) in
-        (match confirmed_result ~with
-        | Some result -> result
-        | None -> (* Transaction is in the transition_frontier *)
-          let block_confirmation_with_ledger_proof_nonces = Option.value_exn (List.filter_map results ~f:(fun (_, `Ledger_proof ledger_proof, `Consensus_status consensus_status_id) ->
-            match Consensus_status.of_int status with
-            | Pending block_confirmation ->  Some (block_confirmation, ledger_proof_nonce)
-            | _ -> None))
-          in
-          let (arg_max_ledger_proof_nonce, max_block_confirmation) = List.fold block_confirmation_with_ledger_proof_nonces ~f:(fun ((curr_max_confirmations, _) as acc) (confirmations, ledger_proof_nonce) ->
-          if confirmations > curr_max_confirmations then
-            (confirmations, ledger_proof_nonce) else acc
-          ) in
-          (is_transaction_snarked best_tip_breadcrumb ledger_proof_nonce, Pending confirmations)
-      )
-```
-
-### Pagination Queries
-
-We can paginate `blocks` (sorting by `time_seen`) query using the following SQL query
-
-```SQL
-
-SELECT state_hash, transaction_ids, time_seen, creator, .... FROM
-(
-SELECT state_hash, CONCAT(block_to_transactions.transaction_id) as transaction_ids, MIN (block_to_transactions.time_seen_transaction) as time_seen
-FROM block_to_transactions
-GROUPBY block_to_transactions.state_hash
-HAVING ((block_to_transactions.sender IN $SENDERS) OR (block_to_transactions.receiver IN $RECEIVERS))
-) as group_by
-INNER JOIN
-blocks
-ON group_by.state_hash = blocks.state_hash
-WHERE group_by.time_seen > $TIME_SEEN
-ORDER BY time_seen
-LIMIT 10
--- Would need to use a groupby query to make query the transactions as well
-```
-
-### Pooled_user_command
-
-```SQL
--- querying specific receivers from the transaction pool
-SELECT * FROM
-transactions
-WHERE status = ADDED AND to = $RECEIVER
-```
-
-```SQL
--- querying specific senders from the transaction pool
-SELECT * FROM
-transactions
-WHERE status = ADDED AND to = $RECEIVER
-```
+https://www.notion.so/codaprotocol/Proposed-Graphql-Commands-57a434c59bf24b649d8df7b10befa2a0
```

Comment:
notion is private, we want this discussion to be public -- can you print this as a pdf and attach the pdf to this PR if you want to avoid the markdown

Reply:
Comments on the notion:

1. Doesn't `blocks` query need to be on the database not on the wallet?
2. `followUpdate` doesn't this have to be where the filter lives? Doesn't the filter live on the daemon?

Reply:
> notion is private, we want this discussion to be public -- can you print this as a pdf and attach the pdf to this PR if you want to avoid the markdown

Planning to do this. I just wanted to get my draft looked at before I share it to the public.